index.html

```diff
@@ -380,6 +382,34 @@ <h1 class="inter700">I'm always interested in hearing about new projects, so if
         </div>
     </footer>
 
+    <!-- menu-mobile-section -->
+    <nav id="mobile-menu" class="mobile-menu">
+        <div id="close-menu-button">
+            <a href="#"><img src="./icons/closeIcon.svg" alt="close menu" /></a>
+        </div>
+        <div>
+            <ul id="mobile-menu-list">
+                <li class="noTagList">
+                    <a href="#works" class='mobile-menu-link inter700'>
+                        Portfolio
+                    </a>
+                </li>
+                <li class="noTagList">
+                    <a href="#about" class='mobile-menu-link inter700'>
+                        About
+                    </a>
+                </li>
+                <li class="noTagList">
+                    <a href="#contact" class='mobile-menu-link inter700'>
+                        Contact
+                    </a>
```

Comment:
- [ ] Please, make sure each element match the figma design. You can do that by adding an `border-bottom` to each element and add some `padding` to match the figma design. 

This will make your project prettier and make the user know you're an awesome developer as you can precisely follow a professional web developer's design 😃 💪 

**Your design**
![image](https://user-images.githubusercontent.com/74411135/231295583-58158666-289c-4a5d-a740-8a049fc482f1.png)

**Figma design**
![image](https://user-images.githubusercontent.com/74411135/231295395-0978ae71-abaa-4067-8359-a81a72ca7815.png)

Reply:
Thanks for your suggestions. The suggested changes are Done.